tests/test_api.py

```diff
@@ -473,3 +473,73 @@ def test_unenrolls_api(self):
             data['population']['variant'][1],
             {'window': self.window2[0].isoformat(), 'count': 10}
         )
+
+
+class TestEnrollmentCountsApi(EnrollmentBaseTestCase):
+
+    def setUp(self):
+        self.url = reverse('v2-experiment-enrolls', args=['pref-flip-1'])
+        self.create_data()
+
+    def test_enrolls_404(self):
+        response = self.client.get(
+            reverse('v2-experiment-enrolls', args=['foo']))
+        self.assertEqual(response.status_code, 404)
+
+    def test_enrolls_api(self):
+        response = self.client.get(self.url)
+        self.assertEqual(response.status_code, 200)
+        data = response.json()
+        result_date1 = datetime.datetime(self.window1[0].year,
+                                        self.window1[0].month,
+                                        self.window1[0].day,
+                                        self.window1[0].hour,
+                                        0, 0, 0).isoformat()
+        result_date2 = datetime.datetime(self.window2[0].year,
+                                        self.window2[0].month,
+                                        self.window2[0].day,
+                                        self.window2[0].hour,
+                                        0, 0, 0).isoformat()
+        self.assertDictEqual(
+            data['population']['control'][0],
+            {'window': result_date1 + '+00:00', 'count': 30}
+        )
+        self.assertDictEqual(
+            data['population']['variant'][0],
+            {'window': result_date2 + '+00:00', 'count': 32}
+        )
```

Comment:
We may need to not use the `EnrollmentBaseTestCase` for this since it is using test data with time stamps via `now()`, which won't guarantee consistent counts if we aggregate per hour. I would suggest inserting specific records in 5m increments spanning an hour boundary and test that it sums them correctly. That should suffice for a test I would think. Look at the `TestPopulationApi` class for an example.